dspace/bin/dspace.bat

```diff
@@ -39,12 +39,12 @@ REM Otherwise, default Java to using 256MB of memory
 if "%JAVA_OPTS%"=="" set "JAVA_OPTS=-Xmx256m -Dfile.encoding=UTF-8"
 
 REM Execute Java
-java %JAVA_OPTS% -classpath "%DSPACE_CLASSPATH%" org.dspace.app.launcher.ScriptLauncher %*
+java %JAVA_OPTS% -classpath "%DSPACE_CLASSPATH%" -Dlog4j.configurationFile=%DSPACEDIR%/config/log4j2.xml org.dspace.app.launcher.ScriptLauncher %*
```

Comment:
Is there a way to not "hardcode" this into our commandline scripts, and simply read it from `log.init.config` setting in our dspace.cfg?

Also, not sure if this change will work if `%DSPACEDIR%` has spaces...we may need to surround the path with quotes (like `-classpath` is).